_search-plugins/knn/settings.md

```diff
@@ -23,3 +23,4 @@ Setting | Default | Description
 `knn.plugin.enabled`| true | Enables or disables the k-NN plugin.
 `knn.model.index.number_of_shards`| 1 | Number of shards to use for the model system index, the OpenSearch index that stores the models used for Approximate k-NN Search.
 `knn.model.index.number_of_replicas`| 1 | Number of replica shards to use for the model system index. Generally, in a multi-node cluster, this should be at least 1 to increase stability.
+`knn.advanced.filtered_exact_search_threshold`| null | Threshold value for the filtered IDs that is used to switch to exact search during filtered ANN search. If number of filtered IDs in a segment is less than this setting's value, exact search will be done on the filtered IDs.  
```

Comment:
Here we use "ANN", above we use "Approximate k-NN Search", and earlier we spelled out "approximate nearest neighbor". Please ensure that we define ANN on first appearance and use only the abbreviation thereafter.